codice/alliance
# CAL-210 Add ability to create a NITF chip

#### What does this PR do?

Add the ability to create a NITF chip of a NITF image.  

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@dcruver 
@rzwiefel 
@bdeining 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire
#### How should this be tested?

In order to build this code, you must first build and install imaging-nitf 0.7-SNAPSHOT using https://github.com/codice/imaging-nitf/pull/146.

Ingest a NITF image. Search for the image in the catalog, and then select the "Actions" menu. Select the "Chip Image" action item. Select a subsection of the image (I suggest that you pick a geographically unique section to make verification easier). At the bottom of the chipping window, select 'download as nitf'. Your browser should prompt you to download/save. The file will be downloaded. Find the image in your download directory and renaming it with a ".ntf" file extension. Ingest the chip. Search the catalog again. You should be able to see the geographic boundaries of the original image and the chipped image on the globe. Visually verify that the geographic boundary drawn for the chipped is approximately the area you selected when creating the chip. Because of precision limitations in how NITF stores the geographic information, the on-screen geographic boundary may not be exact, especially for very small geographic areas.

#### Any background context you want to provide?
#### What are the relevant tickets?

[CAL-210](https://codice.atlassian.net/browse/CAL-210)
#### Screenshots (if appropriate)
#### Checklist:
